### ISSUE TITLE
Service worker assets group installMode: lazy updateMode: lazy doesn't work as expected

### ISSUE THREAD

**Issue by Serginho** [angular/angular#25283]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
[ ] Other... Please describe:
</code></pre>

## Current behavior
With installMode:lazy updateMode: lazy configurations, the service workers downloads all assets.

## Expected behavior
With that configuration. installMode: lazy, I expected the service worker only caches requests of assets that the applications requests. That's the behavior described in docs.

With updateMode: lazy, I expected the service worker update only cached files, and not the whole files.

## Minimal reproduction of the problem with instructions
configure assets in this way:
```
{
      "name": "assetsLazy",
      "installMode": "lazy",
      "updateMode": "lazy",
      "resources": {
        "files": [
          "/assets/**"
        ]
 }
```
Build and run in production, the service worker will caches all assets.

## Environment

<pre><code>
Angular version: 5.2.11

Browser:
- [X] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX 8.9.4
- Platform: Mac
Others:

</code></pre>

**Comment by gkalpak** [comment]
This should have been fixed by #24127. If this is still an issue with the latest version, please open a new issue, providing the necessary details (including a minimal reproduction).

**Comment by Serginho** [comment]
Ok. I'm seeing it was fixed on 6.1.0 Thx
